compiler/luci/pass/src/VerifyQuantizedNodeChannelWiseGranularity.h

```diff
@@ -60,33 +81,43 @@ struct VerifyQuantizedNodeChannelWiseGranularity final : public luci::CircleNode
 private:
   bool visit(const luci::CircleConv2D *node)
   {
+    RETURN_FALSE_UNLESS(is_lwq(node))
```

Comment:
So far, all operators are checked with is_lwq, but I'm not sure this will be true in the future. More specifically, I'm thinking about boolean type or int type (e.g., argmax).